### PR TITLE
Feat: Integrate Swap with FriendlyMarket

### PR DESCRIPTION
**Pull request by hoailinh1210**
### Summary (Please recap what you changed or fixed)
- Integrate Swap feature with FriendlyMarket

### Checklist (If you won't pass this checklist please comment why)

-   [x] I removed all commented or unnecessary code.
-   [x] Provide the screenshots due to UI changed or bug fixed
-   [ ] My code has covered these test cases (please list down your tested cases):

Screenshot 2023-05-26 at 15 16 09

Screenshot 2023-05-26 at 15 15 59

Screenshot 2023-05-26 at 15 15 52

Screenshot 2023-05-26 at 15 18 07

### CODE REVIEW

File: client/src/apiServices/coingecko/request.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  (response) => response.data,
+  (error) => {
+    const { status } = error.response;
+
+    if (status === 400) {
+      const {
+        data: { message },
+      } = error.response;
+
+      alert(message);
```
Comment (duc-q) [inline client/src/apiServices/coingecko/request.js:19]: shoud we use console.error for this?

File: client/src/components/Common/MiddleTruncatedText/index.jsx [hunk 2]
```diff
@@ -3,13 +3,14 @@ import { Tooltip, OverlayTrigger } from 'react-bootstrap';
 import { getEndString } from '../../../helpers/format';
 import './MiddleTruncatedText.scss';
 
-export const MiddleTruncatedText = ({ children, end, placement = 'top' }) => {
+export const MiddleTruncatedText = ({ children, end, start, placement = 'top' }) => {
```
Comment (duc-q) [inline client/src/components/Common/MiddleTruncatedText/index.jsx:6]: this should be exported as default

File: client/src/components/web-extension/Swap/SwapSettings/index.jsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+                                  {item} %
+                                </div>
+                              </RadioButton>
+                            )
+                          })
+                        }
+                      </RadioButtonGroup>
+                  </div>
+                  <div className="cd_we_swap_settings_slippage_input">
+                    <input type="number" onChange={(e) => handleSlippageChange(e.target.value)} value={swapSettings.slippage}/>
```
Comment (duc-q) [inline client/src/components/web-extension/Swap/SwapSettings/index.jsx:46]: for input type="number", we should have min value

**Review by duc-q — COMMENTED** [review]
Just minor feedback, lgtm

File: client/src/components/Common/MiddleTruncatedText/index.jsx (hunk 2 above)
Comment (hoailinh1210) [reply client/src/components/Common/MiddleTruncatedText/index.jsx:6]: Hi @duc-q , this component used in other components, so I will create another PR for it :D

**Review by djhahe — APPROVED** [review]

**Pull request merged**